src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -9605,10 +9617,15 @@ void Player_Action_8084411C(Player* this, PlayState* play) {
         if (sp3C > 0) {
             func_8083A098(this, GET_PLAYER_ANIM(PLAYER_ANIMGROUP_landing, this->modelAnimType), play);
             this->skelAnime.endFrame = 8.0f;
+
+            // `func_8083A098` above can choose from a few different "idle" action variants.
```

Comment:
nit: I think it's a bit confusing to call the other actions there "idle", since the naming of Player_Action_Idle suggests it is the one true idle function, but this could be revisited when func_8083A098/Player_Action_80840450/Player_Action_808407CC are named

Reply:
Yeah im still not sure exactly how we are going to reconcile these variants of the actions. I think parallel and friendly lockon use the same variant, and then "battle" or hostile lockon uses another. Its gonna get real interesting.

I still see this as "true" idle, while the others are variants. Unless we want to specify `UntargetedIdle` later I guess. but bleh

Anyway, agree on reevaluating soon

Reply:
I agree, maybe something like "standing" or "standing still" for the variants that `func_8083A098` chooses from (although idk if link's battle dance counts as that)